src/Maker/MakeMigration.php

```diff
@@ -0,0 +1,102 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\MakerBundle\Maker;
+
+use Doctrine\Bundle\MigrationsBundle\Command\DoctrineCommand;
+use Symfony\Bundle\MakerBundle\ApplicationAwareMakerInterface;
+use Symfony\Bundle\MakerBundle\ConsoleStyle;
+use Symfony\Bundle\MakerBundle\DependencyBuilder;
+use Symfony\Bundle\MakerBundle\ExtraGenerationMakerInterface;
+use Symfony\Bundle\MakerBundle\InputConfiguration;
+use Symfony\Bundle\MakerBundle\MakerInterface;
+use Symfony\Component\Console\Application;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\ArgvInput;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+
+/**
+ * @author Amrouche Hamza <hamza.simperfit@gmail.com>
+ */
+final class MakeMigration implements MakerInterface, ApplicationAwareMakerInterface, ExtraGenerationMakerInterface
+{
+    /**
+     * @var Application
+     */
+    private $application;
+
+    public static function getCommandName(): string
+    {
+        return 'make:migration';
+    }
+
+    public function setApplication(Application $application)
+    {
+        $this->application = $application;
+    }
+
+    public function configureCommand(Command $command, InputConfiguration $inputConf)
+    {
+        $command
+            ->setDescription('Creates a new migration based on database changes.')
+            ->addOption('db', null, InputOption::VALUE_REQUIRED, 'The database connection to use for this command.')
```

Comment:
I'd remove *"...to use for this command"* in all descriptions because it's redundant. Also, I'd like better descriptions. For example, this doesn't help me -> `db = The database connection.` I can't provide a database connection to a console command. Is this the name of the database connection? The name of the Doctrine database connection? etc.